.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 phpunit.xml
 vendor
+.idea
+composer.lock
```

Comment:
Could you please not modify the `.gitignore`?

`.idea` should be in your global `.gitignore`
`composer.lock` should be kept to lock down the versions people use